angular.json

```diff
@@ -9,50 +9,6 @@
       "projectType": "application",
       "prefix": "app",
       "schematics": {
-        "@ngrx/schematics:component": {
-          "styleext": "scss",
-          "spec": false
-        },
-        "@ngrx/schematics:class": {
-          "spec": false
-        },
-        "@ngrx/schematics:directive": {
-          "spec": false
-        },
-        "@ngrx/schematics:guard": {
-          "spec": false
-        },
-        "@ngrx/schematics:module": {
-          "spec": false
-        },
-        "@ngrx/schematics:pipe": {
-          "spec": false
-        },
-        "@ngrx/schematics:service": {
-          "spec": false
-        },
-        "@schematics/angular:component": {
-          "styleext": "scss",
```

Comment:
So far, I've not been able to work out how to recreate this config. 

Any attempt to add this results in this error
```
Schema validation failed with the following errors:
  Data path "/schematics/@schematics~1angular:component" must NOT have additional properties(styleext).
```

Reply:
Without this setting, the Cli will generate css files, not scss files. 



`ng generate component test`

results in these changes: 
CREATE src/app/test/test.component.html (19 bytes)
CREATE src/app/test/test.component.spec.ts (612 bytes)
CREATE src/app/test/test.component.ts (267 bytes)
CREATE src/app/test/test.component.css (0 bytes)
UPDATE src/app/app.module.ts (23424 bytes)

Reply:
Turns out that the fix required was a change from "styleext" to "style". 

ng generate will now default to scss